docs/architecture/decisions/0022-submit-domain-request-user-flow

```diff
@@ -0,0 +1,21 @@
+# 22. Submit Domain Request User Flow
+
+Date: 2023-07-18
+
+## Status
+
+Accepted
+
+## Context
+
+Historically, Verisign has managed the identity verification for users who request to apply for a .gov domain. With CISA's new system, any user who creates an account and verifies themselves through Login.gov will be able to request a .gov domain. As another layer of mitigation against abuse of the system, we needed a way to stop new users from submitting multiple domain requests before they are verified by CISA analysts.
+
+## Considered Options
+
+Option 1: Users that don't meet the requirement of having a prior approved application will not be able to submit any new applications. We add a page alert informing the user that they cannot submit their application because they have an application in one of these "3" statuses (Submitted, In Review or Action Needed). They would still be able to create and edit new applications, just not submit them. The benefits of this option are that it would allow users to have multiple applications essentially in "draft mode" that are queued up and ready for submission after they are permitted to submit (after approval of 1 application).
```

Comment:
I would add here that the user can also just have an existing domain, instead of an approved application. You mention this in the decision, so good to have it here too.